internal/migrations/upgrade_repo_v2.go

```diff
@@ -30,16 +34,37 @@ func (UpgradeRepoV2) Check(ctx context.Context, repo restic.Repository) (bool, e
 }
 
 func (UpgradeRepoV2) Apply(ctx context.Context, repo restic.Repository) error {
-	cfg := repo.Config()
-	cfg.Version = 2
+	tempdir, err := os.MkdirTemp("", "restic-migrate-upgrade-repo-v2-")
```

Comment:
Maybe it's time to bump the Go version again? I know you just bumped to 1.15, but even 1.16 isn't getting security fixes anymore.

Reply:
Maybe. Hm. I'd like to look at what Debian supports, because that's a widely used but very conservative (*cough*) distribution. Requiring 1.16 would prevent users who use the Debian packages for Go to compile restic on Debian oldstable and Debian stable (without backports). If we have a good reason to raise the version to 1.16 then I think we can do that regardless.

Reply:
Debian stable still has 1.15: https://packages.debian.org/bullseye/golang-go

Reply:
Yes, so at the moment we can build `master` with it, but we wouldn't if we were to raise the minimum version to 1.16. That's what I meant.